data.sql

```diff
@@ -8,4 +8,25 @@ INSERT INTO animals(id, name, escape_attempts, neutered, weight_kg, date_of_birt
 VALUES (2, 'Gabumon', 2, TRUE, 8, TO_DATE('Nov 15th, 2018', 'Mon DDth, YYYY'));
 
 INSERT INTO animals(id, name,escape_attempts, neutered, weight_kg, date_of_birth)
-VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
\ No newline at end of file
+VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 5, 'Charmander', 0, FALSE, 11,TO_DATE('Feb 8th, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 6, 'Plantmon', 0, TRUE, 5.7,TO_DATE('Nov 15th, 2021', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 7, 'Squirtle', 3, FALSE, 12.13 ,TO_DATE('Apr 2nd, 1993', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 8, 'Angemon', 1, TRUE, 45 ,TO_DATE('Jun 12th, 2005', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 9, 'Boarmon', 7, TRUE, 20.4 ,TO_DATE('Jun 7th, 2005', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 10, 'Blossom', 3, TRUE, 17 ,TO_DATE('Oct 13th, 1998', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 11, 'Ditto', 4, TRUE, 22 ,TO_DATE('May 14th, 2022', 'Mon DDth, YYYY'));
```

Comment:
- [ ] Please make sure that the entered data is the same as the data given in the exercise. Some of the given `weight_kg` values should be negatives. This is intended and not a mistake as they are going to be fixed later with queries.